scripts/globals/caskets.lua

```diff
@@ -171,7 +171,7 @@ local function dropChance(player)
     --end
 
     if player:hasStatusEffect(tpz.effect.PROWESS_CASKET_RATE) then
```

Comment:
Change the call to dropChance(player) to dropChance(mob) and then find where it is called in the file and pass the mob instead.

When we do this correctly, you'll be able to remove this whole if statement and just do prowessCasketsPower = mob:getHighestCR().